README.md

```diff
@@ -536,6 +544,63 @@ Settings:
 * `migrations_folder` migrations files location (Default: `db/migrations/versions`)
 * `adjust_messages` list of alembic adjust messages (Default: `# ### commands auto generated by Alembic - please adjust! ###`, `# ### end Alembic commands ###`)
 
+#### alembic.load-db-dump
+
+Reset db and load db dump.
+
+Uses [downgrade](#alembicdowngrade) and [load-db-dump](#dbload-db-dump)
+
+Requires [python-decouple](https://github.com/HBNetwork/python-decouple)
+
+Installed with `[env_settings]`
+
+Settings:
+
+* `db_config_mapping` Mapping of db config (Default: `{ "dbname": "rds_db_name", "host": "rds_db_host", "port": "rds_db_port", "username": "rds_db_user", "password": "rds_db_password"}`)
```

Comment:
The same as for https://github.com/saritasa-nest/saritasa-invocations/pull/41/files#r1377285437